google-cloud-storage/src/main/java/com/google/cloud/storage/ParallelCompositeUploadBlobWriteSessionConfig.java

```diff
@@ -628,6 +673,79 @@ protected String fmtFields(String randomKey, String nameDigest, String partRange
     }
   }
 
+  /**
+   * A Decorator which is used to manipulate metadata fields, specifically on the part files used in
+   * a Parallel Composite Upload
+   *
+   * @see #withPartMetadataFieldDecorator(PartMetadataFieldDecorator)
+   * @since 2.35.1 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  @Immutable
+  public abstract static class PartMetadataFieldDecorator implements Serializable {
+
+    abstract PartMetadataFieldDecoratorInstance newInstance(Clock clock);
+
+    /**
+     * A decorator that is used to manipulate the Custom Time Metadata field of part files. {@link
+     * com.google.storage.v2.Object#customTime_}
```

Comment:
```suggestion
     * BlobInfo#getCustomTimeOffsetDateTime()}
```

We should link to our model, rather than the protos since our model is what the customer interacts with.